src/js/components/RadioButton/StyledRadioButton.js

```diff
@@ -15,14 +15,23 @@ const StyledRadioButtonContainer = styled.label`
   user-select: none;
   width: fit-content;
   ${props => props.disabled && disabledStyle} ${props =>
-    !props.disabled &&
-    'cursor: pointer;'}
+  !props.disabled && 'cursor: pointer;'}
 
   :hover input:not([disabled]) + div,
   :hover input:not([disabled]) + span {
     border-color: ${props =>
       normalizeColor(props.theme.radioButton.hover.border.color, props.theme)};
   }
+  :hover {
```

Comment:
I think this hover should also only happen when the input is not disabled, similar to the ones above. Not sure off the top of my head what the selector following it would be (div, span, maybe nothing) but could you try that locally?

Reply:
I wasnt sure about that because the behavior in the design-system has hover on disabled so Im not sure if that was what we wanted that was why I left it

Reply:
That might be something we need to fix in the design system theme. I think most grommet components follow the behavior that if a component is disabled, it doesn't receive hover styling.

Reply:
That makes more sense I think your right with that just being an issue on the design-system..